projects/storefrontlib/layout/a11y/keyboard-focus/autofocus/auto-focus.service.ts

```diff
@@ -40,15 +43,18 @@ export class AutoFocusService extends EscapeFocusService {
    * @param group the optional group for the persistent state, to separate different focus
    *   groups and remain the persistence
    */
-  protected getPersisted(host: HTMLElement, group?: string): HTMLElement {
+  protected getPersisted(
+    host?: HTMLElement | null,
```

Comment:
I don't think we should make host optional. Similar reason as [this](https://github.com/SAP/spartacus/pull/15658/files#r870749898)

Reply:
yup actually, this case here is a good one to make it optional instead of adding undefined type to the host param. Referencing cases like the one we spoke about for `set(key: string | undefined, group?: string | null) {`